kommander: Rewrite federated platform services doc and move to workspaces (#3383)
mesosphere/dcos-docs-site: pages/dkp/kommander/1.3/workspaces/platform-service-dependencies/index.md

## Platform Service Dependencies

When deploying or troubleshooting platform services, it helps to understand how platform services interact and may require other platform services as dependencies.

If a platform service’s dependency does not successfully deploy, the platform service requiring that dependency does not successfully deploy. The table below lists workspace platform services and their dependencies.

<p class="message--important"><strong>IMPORTANT: </strong>All platform services are dependent on the Kubeaddons controller, which manages the installation of platform services.</p>

| **Platform Service** | **Dependencies** |
| --- | --- |
| cert-manager |  |
| elasticsearch |  |
| elasticsearch-curator | elasticsearch |
| elasticsearchexporter | elasticsearch |
| fluentbit | elasticsearch |
| kibana |  elasticsearch, traefik |
| kube-oidc-proxy | cert-manager, traefik |
| kubecost | traefik |
| prometheus | traefik |
| prometheusadapter | prometheus |
| reloader |  |
| traefik | cert-manager, reloader |
| traefik-forward-auth | traefik |

Please see the following sections for more information about the workspace platform services.

### Foundational Components

> The foundational components are essential and provides the foundation for all platform service capabilities and deployments on managed clusters. These components must be enabled to ensure that platform services work properly.

Reviewer: ```suggestion
The foundational components are essential and provide the foundation for all platform service capabilities and deployments on managed clusters. You must enable these components to ensure that platform services work correctly.
```
